portfolio/src/main/webapp/fetchChartMap.js

```diff
@@ -0,0 +1,37 @@
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Fetches section data and uses it to create a chart. */
+function drawChart() {
+  fetch('/chart').then(response => response.json())
+  .then((sectionLiking) => {
+    const data = new google.visualization.DataTable();
+    data.addColumn('string', 'section');
+    data.addColumn('number', 'count');
+    Object.keys(sectionLiking).forEach((section) => {
+      data.addRow([section, sectionLiking[section]]);
+    });
+
+    const options = {
+      'title': 'Favorite Section',
+      'width':600,
+      'height':500
+    };
+
+    const chart = new google.visualization.ColumnChart(
+        document.getElementById('chart-container'));
+    chart.draw(data, options);
+  });
+}
+
+// loads map 
+function initMap() {
+  // The location of college
+  var college = {lat: 28.751348, lng: 77.118107};
```

Comment:
use const/let instead of var.
Use an ES6 linter or read about ES6 specifications